### ISSUE TITLE
FasterCache Support for CogVid 1.5

### ISSUE THREAD

**Issue by gitanon112** [Vchitect/FasterCache#8]
Hello,

Does FasterCache work with CogVid1.5 T2V and I2V models? Do we need to make any script changes to do so? Thanks!

**Comment by keyonchrist** [comment]
Amazing work yall, im also wondering if it supports I2V! Please let us know and if not what script mods we could make to make it happen

**Comment by dummyapps** [comment]
yes, it is, the speed is much faster , the consistency is much better, great job!
